vtex-apps/store-components
# Set loadingItem on onSelectItem at SKUSelector.

#### What problem is this solving?
Prevent the user clicks on this button while the new selectedItem is being processed.
To fix it, the onSelectItem handler dispatches an action on ProductContext, to set the loading item true. So, the add- to-cart-button is disabled while loadingItem is true

#### How to test it?
[Workspace](https://brasileiro--melisseiras.myvtex.com/melissa-sun-rodeo-vermelho-33530/p?skuId=135439)

1. Chose another SKU, via SKU selector
2. Click on add to cart button
3. Check if the correct SKU is added to the minicart.

#### Screenshots or example usage:



#### Describe alternatives you've considered, if any.



#### Related to / Depends on
https://github.com/vtex-apps/product-context/pull/61

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/duL28c2tptZ0zAopCf/giphy.gif)
